app/src/main/res/values/strings.xml

```diff
@@ -123,4 +123,14 @@
     <!--HudView-->
     <string name="search_for_location">Waiting for GPS</string>
 
+    <!--MapSettingsFragment-->
+
+    <string name="points_of_interest">Points Of Interest</string>
+    <string name="btnRename">Rename</string>
+    <string name="btnDelete">delete</string>
+    <string name="update_poi">update</string>
+    <string name="delete_poi">delete</string>
+    <string name="point_deleted_successfully">Point deleted successfully.</string>
+    <string name="point_renamed_successfully">Point renamed successfully.</string>
```

Comment:
"Point deleted successfully." is not a sentence, so it also does not need a dot.

Reply:
removed dots